code/modules/mob/living/carbon/human/species/station.dm

```diff
@@ -930,20 +930,23 @@
 	remains_type = /obj/effect/decal/remains/robot
 
 	eyes = "blank_eyes"
-	brute_mod = 2.5 // 100% * 2.5 * 0.6 (robolimbs) ~= 150%
-	burn_mod = 2.5  // So they take 50% extra damage from brute/burn overall.
+	brute_mod = 2.28 // 100% * 2.28 * 0.66 (robolimbs) ~= 150%
+	burn_mod = 2.28  // So they take 50% extra damage from brute/burn overall.
 	tox_mod = 0
 	clone_mod = 0
 	oxy_mod = 0
 	death_message = "gives one shrill beep before falling limp, their monitor flashing blue before completely shutting off..."
 
-	species_traits = list(IS_WHITELISTED, NO_BREATHE, NO_SCAN, NO_BLOOD, NO_PAIN, NO_DNA, RADIMMUNE, VIRUSIMMUNE, NOTRANSSTING)
+	species_traits = list(IS_WHITELISTED, NO_BREATHE, NO_SCAN, NO_INTORGANS, NO_PAIN, NO_DNA, RADIMMUNE, VIRUSIMMUNE, NOTRANSSTING)
```

Comment:
why did `NO_INTORGANS` have to be added again? I know there was a really good reason for it, I just cna't remember.

Reply:
IT was either something with surgery....or somethingelse...

Reply:
Heart attacks.